app/actions/freeze-funds.js

```diff
@@ -0,0 +1,373 @@
+import BN from 'bignumber.js';
+import { CACHE_MAPS, OPERATIONS_IDS } from 'echojs-lib';
+
+import { getOperationFee } from './transaction-actions';
+import { signTransaction } from './sign-actions';
+import {
+	setFormError, setFormValue, setValue,
+} from './form-actions';
+
+import ValidateSendHelper from '../helpers/validate-send-helper';
+import FormatHelper from '../helpers/format-helper';
+
+import {
+	BROADCAST_LIMIT,
+	ECHO_ASSET_ID,
+	EXPIRATION_INFELICITY,
+	GLOBAL_ID_1,
+	FREEZE_FUNDS_PERIODS,
+} from '../constants/global-constants';
+import { FREEZE_FUNDS_KEYS } from '../constants/transaction-constants';
+import { FORM_FREEZE } from '../constants/form-constants';
+import { WALLET } from '../constants/routes-constants';
+
+import GlobalReducer from '../reducers/global-reducer';
+import { history } from '../store/configureStore';
+import Services from '../services';
+
+export const changeAccount = (fromId) => (dispatch) => {
+	dispatch(setFormValue(FORM_FREEZE, 'from', fromId));
+	dispatch(setFormError(FORM_FREEZE, 'fee', ''));
+	dispatch(setFormError(FORM_FREEZE, 'amount', ''));
+};
+
+/**
+ *  @method checkFeePool
+ *
+ * 	Remove balances and assets by deleted user's id
+ *
+ * 	@param {Object} coreAsset
+ * 	@param {Object} asset
+ * 	@param {Number} fee
+ */
+const checkFeePool = (coreAsset, asset, fee) => {
```

Comment:
some methods in this file were copied from app/actions/transfer-actions.js  
i recommend take out them in separate file, which will be general for all transactions